.github/workflows/cdelivery-s3-apps.yml

```diff
@@ -36,6 +36,8 @@ on:
       # Environment variables
       app-id:
         required: true
```

Comment:
Will we need to remove this then ?

Reply:
We actually still need the app id to deploy the app on aws.